test/orange/data/universes.org.json

```diff
@@ -0,0 +1,397 @@
+{
+"_format": "SCALE ORANGE",
+"_version": 0,
+"materials": {
+"cell_to_mat": [
+-1,
+-1,
+-1,
+3,
+4,
+-1,
+0,
+1,
+2
+],
+"names": [
+"media 0",
+"media 1",
+"media 2",
+"media 3",
+"media 4"
+]
+},
+"universes": [
+{
+"_type": "simple unit",
+"bbox": [
+[
+-2.0,
+-6.0,
+-1.0
+],
+[
+8.0,
+4.0,
+2.0
+]
+],
+"cell_names": [
+"[EXTERIOR]",
+"inner_a",
+"inner_b",
+"bobby",
+"johnny"
+],
+"cells": [
+{
+"faces": [
+0,
+1,
+2,
+3,
+4,
+5
+],
+"flags": 1,
+"logic": "0 1 ~ & 2 & 3 ~ & 4 & 5 ~ & ~",
+"num_intersections": 6,
+"zorder": 2
+},
+{
+"faces": [
+6,
+7,
+8,
+9,
+10,
+11
+],
+"logic": "0 1 ~ & 2 & 3 ~ & 4 & 5 ~ &",
+"num_intersections": 6,
+"zorder": 2
+},
+{
+"faces": [
+6,
+7,
+8,
+9,
+11,
+12
+],
+"logic": "0 1 ~ & 2 & 3 ~ & 4 & 5 ~ &",
+"num_intersections": 6,
+"zorder": 2
+},
+{
+"faces": [
+6,
+7,
+9,
+10,
+12,
+13
+],
+"logic": "0 1 ~ & 2 & 3 & 4 ~ & 5 ~ &",
+"num_intersections": 6,
+"zorder": 2
+},
+{
+"faces": [
+0,
+1,
+2,
+3,
+4,
+5,
+6,
+7,
+8,
+9,
+10,
+11,
+12,
+13
+],
+"flags": 1,
+"logic": "0 1 ~ & 2 & 3 ~ & 4 & 5 ~ & 6 7 ~ & 8 & 9 ~ & 10 & 11 ~ & ~ & 6 7 ~ & 8 & 9 ~ & 11 & 12 ~ & ~ & 6 7 ~ & 9 & 10 & 12 ~ & 13 ~ & ~ &",
+"num_intersections": 14,
+"zorder": 2
+}
+],
+"md": {
+"name": "outer",
+"provenance": "universes.org.omn:48"
+},
+"surface_names": [
+"john.mx",
+"john.px",
+"john.my",
+"john.py",
+"john.mz",
+"john.pz",
+"bob.mx",
+"bob.px",
+"inner_a.my",
+"bob.my",
+"bob.mz",
+"inner_a.pz",
+"bob.pz",
+"bob.py"
+],
+"surfaces": {
+"connectivity": [
```

Comment:
Since we now generate connectivity at construction time, can you delete this from the JSON file? And in Shift ORANGE on your branch, can you likewise delete the connectivity from the output routines?